syntax = "proto3";

package proto;

message CellLimits {
  int32 num_x_cells = 1;
  int32 num_y_cells = 2;
}